front/components/assistant/conversation/input_bar/InputBar.tsx

```diff
@@ -157,9 +157,9 @@ export function AssistantInputBar({
     }
 
     const { mentions: rawMentions, text } = textAndMentions;
-    const mentions: MentionType[] = rawMentions.map((m) => ({
-      configurationId: m.id,
-    }));
+    const mentions: MentionType[] = [
+      ...new Set(rawMentions.map((mention) => mention.id)),
```

Comment:
Late to the party but shouldn't we prevent mentioning twice the same assistant in the Input Bar versus only sending one mention when posting the message?

Reply:
I think we should not because UX might we weird for the user:
If there's one mention at the beginning of the message that you don't see, then the message, then you try to mention and it's not working, it's confusing. 

I believe it's better to let them mention as much as they want but summon once per assistant.

Reply:
Not convinced but won't fight for it 🙃

Reply:
For large messages I wanted to keep the usual behavior since users do not necessarily remember that an assistant was mentioned very early in the message. This is totally up to debate tho